### ISSUE TITLE
✨ Support multi line `+ignore` directive

### ISSUE THREAD

**Issue by wingyplus** [dagger/dagger#8383]
### What are you trying to do?

Currently, `+ignore` doesn't support multi-line declaration so I need to declare it on a single line. What I want to purpose is make it support multi-line. For example:

```go
// +ignore=[
//      "**/*",
//      "!generated/", 
//      "!src/", 
//      "!scripts/", 
//      "!composer.json", 
//      "!composer.lock", 
//      "!LICENSE", 
//      "!README.md"
//  ]
```

### Why is this important to you?

Single-line ignore is hard to read when they're grow larger. :(

**Comment by helderco** [comment]
I asked for this here:
- https://github.com/dagger/dagger/pull/7744#issuecomment-2274122714

Feedback:

> > > I need to check how I can parse that but that must be possible.
> 
> Hm, you want `parsePragmaComment`. This is tricky, currently it's all assumed in one line with a regexp.
> 
> Being able to do opening/closing braces means the language is no longer regular, so we won't be able to use regexp for it. I'm not confident there's a simple way to do it (but I'm happy to do it in a follow-up) - I don't think we should delay this PR for that though.

**Comment by wingyplus** [comment]
@helderco Yeah, I open it after discussed with Justin at https://discord.com/channels/707636530424053791/708371226174685314/1282767180311695382.
